### PR TITLE
Docs Updates

### PR DESCRIPTION
**Pull request by collinmcneese**
Signed-off-by: Collin McNeese <collinmcneese@github.com>

- Adds markdownlint and includes with `npm test`
- Updates README and CONTRIBUTING with version2.0 updates

### CODE REVIEW

**Review by sam1el — APPROVED** [review]
Awesome work. Easy to follow. LGTM.

**Review by rockin-robinson — COMMENTED** [review]
👋 Good stuff! I left a couple of comments. I'm happy to make the changes if needed 🥳

File: README.md [hunk 1]
```diff
@@ -1,115 +1,132 @@
-# GitHub Issues to Totango V2
+# GitHub Issues to Totango
 
-This action takes issues in GitHub that have been labeled either 'task' or 'touchpoint' and turns them into tasks or touchpoints in totango accordingly.
+This repository contains a GitHub Action which can be used with a repository to automatically create and `task` and `touchpoint` data types in Totango with GitHub Issues.
```
[the remark this answers is not in the document]
Comment (collinmcneese) [reply README.md]: whoops, good catch!

File: README.md [hunk 2]
```diff
@@ -1,115 +1,132 @@
...
 
-These need to all be set up as secrets in the GitHub Repo where the action will be running.
+This action should be used in a repository in combination with Issues, triggering from `issues` event types.
 
-### `ACCOUNT_ID`
+### Action Inputs
 
-**Required** The customer account id.
-> Browse to the Account Overview page in Totango > Click All to see all the data > Scroll down to the Account Identifier section. This is also available in the URL.
+The following `inputs` are used by this Action.  These values should be available as secrets to the repository which runs this Action.
```
[the remark this answers is not in the document]
Comment (collinmcneese) [reply README.md]: Good catch again, I updated the items there but didn't update this, all set now!

**Review by rockin-robinson — APPROVED** [review]
Sweet! Looks great! Merging!

**Pull request merged**